### PR TITLE
Added functions to get basic content information from SWH API

### PR DESCRIPTION
**Pull request by sanketvarpe**
#### Task completed:

- Created a function and its tests to get content information about any project using SWH api which requires hash type and hash value to fetch data
Signed-off-by: Sanket Varpe <varpesanket2000@gmail.com>

### CODE REVIEW

**Comment by sanketvarpe** [comment]
@pombredanne #1  was little messed up so i have created this new #2  where all commits are boiled down to single commit and also with dco signoff and as we discussed regardinfg those checks which are failing can be taken care of later,so i guess this pr is okay to go with for merging so that i can work on next set of functions.

Thank You. :)

File: setup.cfg [hunk 1]
```diff
@@ -29,7 +29,9 @@ package_dir=
 packages=find:
 include_package_data = true
 zip_safe = false
-install_requires =
+install_requires = 
+    requests
+    requests_mock
```
Comment (pombredanne) [inline setup.cfg]:
```suggestion
```

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+requests
+requests_mock
```
Comment (pombredanne) [inline requirements.txt]: This is likely only needed once in the dev file above and not used except in tests
```suggestion
```

File: requirements-dev.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+requests
```
Comment (pombredanne) [inline requirements-dev.txt]: This is already in the main requirements file below so no need to repeat for dev:

```suggestion
```

**Review by pombredanne — APPROVED** [review]
Thanks!
A few things for your consideration:
- we can remove the .travis.yml file that came from the skeleton. We no longer use Travis for CI
- the setup and skeleton is out of date. I merged it. We should be green and passing all tests anytime now!

**Comment by sanketvarpe** [comment]
> Thanks! A few things for your consideration:
> 
> * we can remove the .travis.yml file that came from the skeleton. We no longer use Travis for CI
> * the setup and skeleton is out of date. I merged it. We should be green and passing all tests anytime now!

Yeah it did passed!!. What was the issue then??Was is due to outdated ci/cd scripts in outdated skeleton.

Thanks :)

**Comment by pombredanne** [comment]
> What was the issue then??Was is due to outdated ci/cd scripts in outdated skeleton.

Yes outdated Ci/CD scripts were the issue!

**Pull request merged**